lib/map/security_token/configuration.rb

```diff
@@ -34,6 +34,10 @@ def client_cert_path
         Settings.map_services.client_cert_path
       end
 
+      def provider_jwks_path
+        Settings.map_services.provider_jwks_path
```

Comment:
I think if this is not likely to change we could just hard code it here, like how `token_path` is hard-coded